### PR TITLE
move env vars out of `pkg`

### PR DESCRIPTION
**Pull request by dekkagaijin**
Signed-off-by: Jake Sanders <jsand@google.com>

### CODE REVIEW

File: cmd/cosign/cli/util.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	"strconv"
+)
+
+const (
+	ExperimentalEnv = "COSIGN_EXPERIMENTAL"
+	ServerEnv       = "REKOR_SERVER"
+	rekorServer     = "https://rekor.sigstore.dev"
+)
+
+func EnableExperimentlal() bool {
```
Comment (dlorenc) [inline cmd/cosign/cli/util.go:28]: nit: typo :)

Comment (dekkagaijin) [reply cmd/cosign/cli/util.go]: gosh dangit @jonjohnsonjr droppin the ball

**Review by dlorenc — APPROVED** [review]

File: cmd/cosign/cli/util.go (hunk 1 above)
Comment (jonjohnsonjr) [reply cmd/cosign/cli/util.go]: if you want me to catch your typos you have to put me as a reviewer

Comment (dekkagaijin) [reply cmd/cosign/cli/util.go]: I thought you could just hear them, like the world's worst superpower

**Pull request merged**